src/styles/price.scss

```diff
@@ -0,0 +1,19 @@
+.price {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 24px;
+  padding-left: $padding-left;
```

Comment:
The variable `$padding-left` is used but not defined. Please define this variable to avoid compilation errors.